eox_tagging/validators.py

```diff
@@ -0,0 +1,68 @@
+""" File to define validations for tag model fields
+"""
+
+import re
+
+from django.conf import settings as base_settings
+from django.core.exceptions import ValidationError
+
+
+class Validators(object):
+    """ Defines all validator methods.
+    """
+
+    def __init__(self, instance):
+        """
+        Attributes:
+            instance: instance of the model to validate before saving
+        """
+        self.instance = instance
+        self.functions = {  # Functions defined for validation
+            "definition": self.__validate_field_definition
+        }
+
+    def run_validators(self):
+        """
+        Runs defined validators
+        """
+        self.__validate_not_update()
+
+        definitions = base_settings.EOX_TAGGING_DEFINITIONS
+
+        for obj in definitions:
+            validations = obj["validations"]
+            for val in validations:
+                self.functions[val](obj)
+
+    def __validate_not_update(self):
```

Comment:
if I press save in the admin panel I will get a 500 error ?

Reply:
Yes. I think I solved it adding an empty EOX_TAGGING_DEFINITIONS to common